tests/docker-compose.yml

```diff
@@ -29,3 +30,18 @@ services:
       /usr/bin/mc mb myminio/floorist;
       /usr/bin/mc policy set download myminio/floorist;
       exit 0;'
+  setupdb:
+    image: registry.access.redhat.com/ubi9/ubi-minimal
+    depends_on:
+      - db
+    links:
+      - db
+    environment:
+      - PGHOST=db
+      - PGDATABASE=floorist
+      - PGUSER=postgres
+      - PGPASSWORD=flooristadmin
+    command: /bin/sh -c '
+      microdnf install -y postgresql;
+      psql -U postgres -c "CREATE EXTENSION IF NOT EXISTS \"uuid-ossp\";";
```

Comment:
Needs to be included in the `run-tests.sh` @Victoremepunto can you help me pls?

Reply:
I need to test this first, but , I think this just needs a `.sql` script added to the DB so it configures the extension. check: https://github.com/sclorg/postgresql-container/tree/generated/12

see https://github.com/sclorg/postgresql-container/tree/generated/12#postgresql-start

I'll try to check myself later, but if you wanna explore it first , please do :+1:

Reply:
I guess we can tackle this in #31